usr.sbin/adduser/adduser.8

```diff
@@ -295,6 +298,19 @@ In addition, it will be available for inclusion in the message file in the
 .Va randompass
 variable.
 .El
+.It Fl z Ar dataset
+Create home directory on ZFS 
+.Ar dataset .
+.It Fl Z
+Set ZFS creation flag to yes.  The
```

Comment:
The default behavior should be to create ZFS dataset unless 1) the `homeprefix` is not a ZFS, or 2) administrator explicitly says no by specifying a command line option, or by an option set in `adduser.conf(5)`.